src/useFormState.js

```diff
@@ -105,7 +108,7 @@ export default function useFormState(initialState, options) {
          * returning the value of input from the current form state is illogical
          */
         if (isCheckbox || isRadio) {
-          return ownValue;
+          return toString(ownValue);
```

Comment:
Per [spec](https://www.w3.org/TR/2010/WD-html5-20101019/the-input-element.html#value-sanitization-algorithm
):

> the user agent must set the value of the element to the value of the value content attribute, if there is one, or the empty string otherwise


Therefore, if `ownValue` isn't specified by the user, it should return an empty string. Previously, it was returning `undefined`. This should not result in any breaking changes.